uint8 OperationMode_Standby = 0
uint8 OperationMode_Pickup = 1
uint8 OperationMode_DropOff = 2
Header header 
uint8 State
uint8 StateUpdate
uint8 OperationMode
